docs/concepts/what-is-a-retry-policy.md

```diff
@@ -85,14 +68,35 @@ Non-Retryable Errors = []
 - **Use case**: Use this attribute to ensure that retries do not continue indefinitely.
   However, in the majority of cases, we recommend relying on the Workflow Execution Timeout, in the case of [Workflows](#workflow), or Schedule-To-Close Timeout, in the case of [Activities](#activity), to limit the total duration of retries instead of using this attribute.
 
-### Non-Retryable Errors
+#### Non-Retryable Errors
 
 - **Description**: Specifies errors that shouldn't be retried.
   - **Default is none.**
   - If one of those errors occurs, the [Activity Task Execution](#activity-task-execution) or [Workflow Execution](#workflow-execution) is not retried.
 - **Use case**: There may be errors that you know of that should not trigger a retry.
   In this case you can specify them such that if they occur, the given execution will not be retried.
 
+### Retry interval
+
+The wait time before a retry is the _retry interval_. A retry interval is the smaller of two values:
+
+- The [Initial Interval](#initial-interval) multiplied by the [Backoff Coefficient](#backoff-coefficient) raised to the power of the number of retries.
+- The [Maximum Interval](#maximum-interval).
+
+### Event History
+
+There are some subtle nuances to how Events are recorded to an Event History when a Retry Policy comes into play.
+
+- For an Activity Execution, the [ActivityTaskStarted](/docs/concepts/what-is-an-event#activitytaskstarted) Event will not show up in the Workflow Execution Event History until the Activity Execution has completed or failed (having exhausted all retries).
+  This is to avoid filling the Event History with noise.
+  Use the Describe API to get a pending Activity Execution's attempt count.
+
+- For a Workflow Execution with a Retry Policy, if the Workflow Execution fails, the Workflow Execution will [Continue-As-New](/docs/concepts/what-is-continue-as-new) and the associated Event is written to the Event History.
+  The [WorkflowExecutionContinuedAsNew](/docs/concepts/what-is-an-event#workflowexecutioncontinuedasnew) Event will have an "initiator" field that will specify the Retry Policy as the value and the new Run Id for the next retry attempt.
+  The new Workflow Execution is created immediately.
+  But the first Workflow Task won't be scheduled until the backoff duration is exhausted.
+  That duration is recorded as the `firstWorkflowTaskBackoff` field of the new run's `WorkflowExecutionStartedEventAttributes` event.
+
 **Implementation guides:**
 
 - [How to set a Retry Policy for a Workflow Execution in Go](/docs/go/how-to-set-startworkflowoptions-in-go/#retrypolicy)
```

Comment:
I think we should move links to implementation guides to just after the intro paragraph.